root/components/list/ReleaseGroupList.js

```diff
@@ -64,8 +71,8 @@ export const ReleaseGroupListTable = withCatalystContext(({
         ? defineSeriesNumberColumn(seriesItemNumbers)
         : null;
       const yearColumn = defineTextColumn(
```

Comment:
Re: my last comment in the resolved thread

```suggestion
      const yearColumn = defineTextColumn<ReleaseGroupT>(
```